### ISSUE TITLE
Not able to auto login on index page

### ISSUE THREAD

**Issue by atif92** [mefernandez/spring-multi-module-oauth-sso#2]
How to make it auto login on index page instead of hitting login on client 2? just like you login to google and on gmail you just see logged in by only hitting refresh on their public page?

**Comment by mefernandez** [comment]
Hi Atif, when logged in, ¿can you enter client2 `private.html`?

**Comment by atif92** [comment]
Yes I can but I want to login using index.html. Is that possible? Because i've seen gmail, youtube and google logging in using public pages?

**Comment by atif92** [comment]
Please help me with this I really want my client 2 to login from index.html just my refresh if my client 1 is already logged in.
